src/olympia/reviewers/tests/test_views.py

```diff
@@ -6852,40 +6875,6 @@ def test_clear_pending_rejections(self):
             version__addon=self.addon, pending_content_rejection__isnull=False
         ).exists()
 
-    def test_clear_pending_rejections_triggers_reset_due_date(self):
```

Comment:
Updating due dates directly when clearing pending rejection date no longer makes sense since it doesn't affect the individual NHR instances anyway.